Sources/GameMP/Controls.cpp

```diff
@@ -227,8 +227,9 @@ void CControls::Load_t( CTFileName fnFile)
       achrActionName[ 0] = 0;
       FLOAT fSensitivity = 50;
       FLOAT fDeadZone = 0;
+      // FIXME DG: gcc and clang say the followign has too many arguments for the given format
       sscanf( achrLine, "%*[^\"]\"%1024[^\"]\"%*[^\"]\"%1024[^\"]\" %g %g %1024s %1024s",
-              achrActionName, achrAxis, &fSensitivity, &fDeadZone, achrIfInverted, achrIfRelative, achrIfSmooth);
+              achrActionName, achrAxis, &fSensitivity, &fDeadZone,   achrIfInverted, achrIfRelative, achrIfSmooth);
```

Comment:
I couldn't parse that string myself, so I was hoping someone else would for me.  :)